src/index.tsx

```diff
@@ -60,6 +60,10 @@ root.render(
   <RMWCProvider
     // Globally disable ripples
     ripple={false}
+    typography={{
+      /** Make headline5 <h5>  */
```

Comment:
How does this comment match the code below?

Reply:
Fixed!